### DISCUSSION TITLE
Plans to provide also Play Store?

### DISCUSSION THREAD

**Discussion by NilsRo** [microsoft/WSA#518] [General]
Tried to use Amazons Store but all Apps even popular like EBay was not available. Now with Play Store implemented via scripts sold all tablets and keep only two Windows Tablets. But like to see Play Store supported officially as there is e.g. no multiuser support and would be much easier. Or at least a simple way to add it anyway...hopefully there are still plans as WSA and WSL are great!

**Comment by Wduran84** [comment]
check out this site it has a version of WSA that uses Google Play Store

**Reply by NilsRo** [reply]
There is no site linked. But I still installed it without Amazon Store as it had mostly senseless Apps. Playstore works great but should be supported by Microsoft @msftgits as well. Would be also Ok to pay for WSA as well, the integration is nice.

**Comment by vonDubenshire** [comment]
I've always used it with Play Store since 2021....


> There are more trustworthy developers providing WSA with Magisk rooted for full control and any Google apps or lack thereof you want. 
> 
> Just search for it I don't want to get in trouble on the official GitHub from MS here
